scripts/bulk_de/generate_pseudobulk.R

```diff
@@ -26,7 +31,7 @@ rownames(filtered_pseudobulk) <- names(sums)
 colnames(filtered_pseudobulk) <- samples
 rm(i, sce, counts, sums); gc()
 
-pseudobulk_path <- paste(local_data_path, "pseudobulk_objects", sep = "/")
+pseudobulk_path <- paste(local_data_path, "sce_objects", sep = "/")
```

Comment:
I'm pretty sure this change was intentional, I just wanted to double check since it's called pseudobulk_path but points to sce data

Reply:
Yeah, I had planned on having a separate folder for pseudobulk data but then all the other pseudobulks went somewhere else, and I kept looking for this file in sce_objects anyway :joy: so I consolidated